### ISSUE TITLE
Bug on Mutiples Reports

### ISSUE THREAD

**Issue by cosminwindox** [Qbox-project/qbx_adminmenu#61]
### Summary

Basically, with more than one report open, if I close one, the other disappears. Then, if I create another one, the previously open one reappears. If I delete it again and then try to create a new one, the report command and menu are completely broken.

### Reproduction

1- Create more than one report.
2 - Delete one of the reports.
3 - Can't use the reports anymore.
4 - Create another report.
5 - Now there are two reports.
6 - Delete another report.
7 - The report command and menu are completely broken.

error on video:
https://youtu.be/aILXchaZ2jM

error on /report
![image](https://github.com/Qbox-project/qbx_adminmenu/assets/45999317/ca5e485e-0af6-4acd-8489-10041b20a940)

### Expected behavior

It was expected that after deleting one report, the other open reports would continue to appear in the menu.

### Actual behavior

Basically, with more than one report open, if I close one, the other disappears. Then, if I create another one, the previously open one reappears. If I delete it again and then try to create a new one, the report command and menu are completely broken.

### Current Version

version '1.0.0'

### Custom Resources

I don't have any custom resources that interfere with qbx_adminmenu.

**Comment by KostaZx** [comment]
@Scullyy
